### ISSUE TITLE
Decide which news/events pages to implement, drop, or delete

### ISSUE THREAD

**Issue by NickSto** [galaxyproject/galaxy-hub#1607]
The [comment](/galaxyproject/galaxy-hub/issues/1480#issuecomment-1170362963) on issue #1480 lists a bunch of types of pages that .eu has beyond the [middle panes](1380) and subsite-specific news/events pages. Are all these really necessary to implement? How many of these are actually used?

I collected analytics data for each of these and compiled them in a spreadsheet, then annotated with whether I think we should keep or discard each page. The public version without analytics data is [here](https://docs.google.com/spreadsheets/d/1GQNoRWam-ujd4iN-CAHHDqnrZBv6bqseNxtXdc26Jqo/edit?usp=sharing) and the restricted version with analytics data (and more formatting) is [here](https://docs.google.com/spreadsheets/d/1wJwI6T98KzaAAaob5m_sUAWx-qP623C4RlMv2-3qoqc/edit?usp=sharing) (the core people should have access but message me if you don't).

Basically, I figure let's keep anything with any traffic in the past year, as long as we've already implemented it. And let's keep the `/[subsite]/news/` and `/[subsite]/events/` pages on principle, even if they didn't get any traffic. What this would mean is we don't have to implement any of the `gxnews.html` or `gxevents.html` pages, and we'll delete all of the subsite middle panes (`/bare/[subsite]/galaxy/`), since it doesn't look like anyone actually uses them.

**Comment by NickSto** [comment]
With #1608 we have one solution. I'm closing unless there's a reason to revisit.
